src/meta/src/barrier/mod.rs

```diff
@@ -835,6 +835,12 @@ impl GlobalBarrierManagerContext {
         let duration_sec = enqueue_time.stop_and_record();
         self.report_complete_event(duration_sec, &command_ctx);
         wait_commit_timer.observe_duration();
+        self.metrics.last_committed_barrier_time.set(
+            std::time::SystemTime::now()
+                .duration_since(std::time::UNIX_EPOCH)
+                .unwrap()
+                .as_secs() as i64,
```

Comment:
You are right. I think it can be renamed to `last_barrier_commit_time`

Reply:
Well, strictly speacking, `last_committed_barrier_commit_time`

Reply:
🤔 So for "barrier pending time", maybe we should get the first uncommitted barrier's epoch time. Current metrics is actually "how long there's no new barrier get commited"

Reply:
+1 for using epoch time. 

> maybe we should get the first uncommitted barrier's epoch time

I think it should be the last committed barrier's epoch time ? If this lags behind, we will know the barrier can't catch up with physical time.

Reply:
I don't fully get your idea. If so, this metrics is like "how long there's no new barrier get committed" + barrier latency?

My original idea is just "how long there's no new barrier get committed", so that we can now the system is stuck

Reply:
Oh, I think your idea represents more "freshness".